### PR TITLE
Add styling for new dynamic barriers

### PR DESCRIPTION
**Pull request by remybach**
## Link to Ticket / Card:

https://trello.com/c/DnyF0Mia/1262-5-dynamic-barriers-styling

## Screenshots:

|Desktop|Mobile|
|-|-|
|||

🐿 v2.12.3

### CODE REVIEW

File: scss/_barrier.scss [hunk 1]
```diff
@@ -163,6 +164,156 @@
...
+			padding-bottom: 60px;
+			position: absolute;
+			width: 1px;
+
+			@include oGridRespondTo(M) {
+				left: 25px;
+			}
+		}
+
+		ul,
```
Comment (liamkeaton) [inline scss/_barrier.scss]: Should these have specific classes incase there are ever any other list styles needed?

Comment (remybach) [reply scss/_barrier.scss]: This was intentional... these will be specified in the config as follows:

```js
brexit: {
	...,
	content:`<ul>
		<li>Unmatched number of scoops</li>
		<li>Regional perspectives</li>
		<li>Brexit briefing newsletter</li>
	</ul>`
}
```

File: scss/_barrier.scss [hunk 2]
```diff
@@ -163,6 +164,156 @@
...
+			content: '';
+			display: block;
+			height: 30px;
+			position: absolute;
+			width: 100%;
+			z-index: 3;
+		}
+
+		.barrier__leading-text {
+			@include oTypographySansBold($scale: -1);
```
Comment (liamkeaton) [inline scss/_barrier.scss]: The origami team said be careful with `oTypographySansBold` and `oTypographySerifBold` mixins as they can duplicate lots of styles, should these be generic modifiers?

**Comment by jkerr321** [comment]
Just from eyeballing, the main heading in the implementation looks a bit squashed compared to the design:

**Implementation:**

image


**Design:**

image

File: scss/_barrier.scss (hunk 2 above)
Comment (remybach) [reply scss/_barrier.scss]: I did try to go without having to use these but for some reason couldn't get it to work. Will have another crack at it now that all the rest of it has fallen into place.

**Comment by remybach** [comment]
The `line-height` does look off... good spot @jkerr321.

FYI, The font is different because FinancierText has been removed from `o-fonts` and we apparently use Georgia now. ¯\\\_(ツ)_/¯

File: scss/_barrier.scss (hunk 2 above)
Comment (remybach) [reply scss/_barrier.scss]: I've had a look and anytime I'm setting the font with this, it's the first time in the hierarchy it's being set. Any changes after that happen with the more specific mixins.

Comment (liamkeaton) [reply scss/_barrier.scss]: 👍 Cool, then there should be no duplications

**Comment by liamkeaton** [comment]
@remybach Financier is still in the registry for o-fonts https://registry.origami.ft.com/components/o-fonts@3.3.2#demo-financier-display

**Comment by remybach** [comment]
@liamkeaton: https://github.com/Financial-Times/o-typography#migrating-from-v4-to-v5

> removing access to the `FinancierText` font family and replace serif font with `Georgia`.

**Comment by liamkeaton** [comment]
`FinancierText` is removed but it still uses `FinancierDisplayWeb` which I believe this is?

**Comment by remybach** [comment]
@liamkeaton ¯\\\_(ツ)_/¯ All I'm doing is using the serif mixin and when I inspect the page I see `font-family: Georgia,serif;`: https://github.com/Financial-Times/n-barriers/pull/162/files#diff-22074eb434b058e0237c39697b1781c9R212

**Comment by liamkeaton** [comment]
Seems strange as the demos on the registry are using `FinancierDisplayWeb`

Screenshot 2019-05-31 at 16 35 48

Gonna check it out and see if I can see anything, might be worth raising with origami?

**Comment by remybach** [comment]
I think there's a distinction here between `o-typography` and `o-fonts`... If you'll excuse me for summoning you @rowanmanning, could you please provide some clarity on the above discussion?

**Comment by liamkeaton** [comment]
That screenshot was from the `o-typography` page

**Comment by notlee** [comment]
Hey 👋 we use `o-fonts` to make fonts available, it includes font faces and can be used to check for available fonts. `o-typography` sets typographic properties for available fonts like `font-family`, `font-size`, and `line-height`.

We use FinancierDisplayWeb for our "display" font `oTypographyDisplay`, which is used for editorial-style headings; Georgia is our "serif" font  `oTypographySerif` used for editorial-style body copy; and MetricWeb is as our "sans" font `oTypographySans` used for most other things. 

Designs should conform to the [uxd toolkit](https://sites.google.com/ft.com/ft-design-system#h.p_K43tV1dcQcbL) and be achievable with [o-typography Sass](https://registry.origami.ft.com/components/o-typography@5.11.6/readme?brand=master#sass). If that's not the case I'm happy to come chat with the team to answer any questions or iron out any divergences :)

**Comment by remybach** [comment]
@notlee That's super helpful... thanks! I wasn't aware of `oTypographyDisplay` and is likely exactly what we were after 🎉

**Comment by remybach** [comment]
@liamkeaton / @jkerr321: I've updated the styles and attached updated screenshots ☝️.

**Comment by liamkeaton** [comment]
Looks good, I _think_ the font weight might need to be a little heavier but I'll leave that for @lucindamaria123's review

**Comment by remybach** [comment]
I've already run it past her and this was her preference. 😉

**Review by liamkeaton — APPROVED** [review]
👍

**Pull request merged**